### DISCUSSION TITLE
Switching from Claude to ChatGpt 4o, Where is the promised ChatGpt 4o?

### DISCUSSION THREAD

**Discussion by GiancarloCubeta** [gitbito/bitoai#223] [Q&A] [answered]
Hi,
I have chosen Bito to work with ChatGpt 4o in my local context awaring, I have got the Advanced AI plan, but the model used is answering my prompts is just Claude and apparently there is no way to switch to ChatGpt 4o as desired. I'm a bit disappointed.

**Comment by amargoel98** [comment, marked as answer]
Hi, we recently switched over to Claude Sonnet 3.5 (from GPT-4o) as we found it is better at coding related tasks and many users asked us to do this. It is on our roadmap to allow you to switch between different models as you wish. 

We have many other great features planned for this quarter, I think you will really like them.
